### ISSUE TITLE
After the fine-tuning is SamAutomaticMaskGenerator applicable

### ISSUE THREAD

**Issue by MustafaArikan** [ziqi-jin/finetune-anything#30]
Hi,

very interesting work. After fine-tuning is the SamAutomaticMaskGenerator applicable
to generate masks similar to the original SAM.

thx

**Comment by ziqi-jin** [comment]
The mask of SAM is an instance segmentation mask without category mark. The instance segmentation task that will be supported later will output the same instance segmentation mask as SAM, and there will be a separate field to store category information. For other tasks, such as matting, semantic segmentation, etc., the mask required by the corresponding task will be output
